### PR TITLE
French translation of Intermediate Front End Development Projects

### LINKED ISSUE #7639: French translation of Intermediate Front End Development Projects
- [ ] Build a Random Quote Machine 
- [ ] Show the Local Weather 
- [ ] Build a Wikipedia Viewer 
- [ ] Use the Twitch.tv JSON API

### PR DESCRIPTION
**Pull request by elazzabi**
Closes #7639

### CODE REVIEW

**Comment by raisedadead** [comment]
@elazzabi It would be great if you can get this reviewed by a peer. 
Refer : https://gitter.im/FreeCodeCamp/TranslationFrench

**Comment by elazzabi** [comment]
Already asked in the chatroom. Still waiting fir someone to check :)

**Comment by theflametrooper** [comment]
@elazzabi We will merge within a week if no can check it. Thanks for your contribution!

**Comment by kaunglay** [comment]
https://gitter.im/FreeCodeCamp/TranslationFrench

**Comment by kaunglay** [comment]
https://gitter.im/FreeCodeCamp/TranslationFrench

**Comment by kaunglay** [comment]
https://gitter.im/FreeCodeCamp/TranslationFrench

**Comment by kaunglay** [comment]
#8725

File: seed/challenges/01-front-end-development-certification/intermediate-ziplines.json [hunk 1]
```diff
@@ -97,9 +108,22 @@
         "Cuando hayas terminado, pulsa el botón de \"I've completed this challenge\" e incluye un enlace a tu CodePen.",
         "Puedes obtener retroalimentación sobre tu proyecto por parte de otros campistas, compartiendolo en nuestra <a href='//gitter.im/freecodecamp/codereview' target='_blank'>Sala de chat para revisión de código</a>. También puedes compartirlo en Twitter y en el campamento de tu ciudad (en Facebook)."
       ],
+      "descriptionFr": [
+        "<strong>Objectif:</strong> Créer une application sur <a href='http://codepen.io' target='_blank'>CodePen.io</a> qui a un fonctionnement similaire à: <a href='http://codepen.io/FreeCodeCamp/full/bELRjV' target='_blank'>http://codepen.io/FreeCodeCamp/full/bELRjV</a>.",
```
Comment (raisedadead) [inline seed/challenges/01-front-end-development-certification/intermediate-ziplines.json]: make the `http` links to `https`

File: seed/challenges/01-front-end-development-certification/intermediate-ziplines.json [hunk 2]
```diff
@@ -97,9 +108,22 @@
...
+        "<strong>Rule #2:</strong> Essayez de répondre aux <a href='https://fr.wikipedia.org/wiki/Récit_utilisateur' target='_blank'>récits d’utilisateurs</a> suivants. Utilisez n’importe quel librairie ou API en ajoutant votre style personnel.",
+        "<strong>Récit d’utilisateur:</strong> Je peux voir la météo de ma région.",
+        "<strong>Récit d’utilisateur:</strong> Je peux voir une icône ou un arrière-plan qui dépendent de la météo (neige, déserts… par exemple).",
+        "<strong> Récit d’utilisateur:</strong> Je peux cliquer sur un bouton pour basculer entre le Celsius et le Fahrenheit.",
+        "On recommande l’utilisation de  <a href='http://openweathermap.org/current#geo' target='_blank'>Open Weather API</a>.  L’utilisation de ce service nécessite la création d’un compte gratuit pour avoir une clé d’utilisation de l’API. Normalement, il faut éviter d’inclure des clés d’API sur CodePen, mais on n’a pas pu trouver un API de météo qui ne nécessite pas une clé.",
+        "N’oubliez pas d’utiliser la méthode  <a href='//github.com/FreeCodeCamp/freecodecamp/wiki/How-to-get-help-when-you-get-stuck' target='_blank'>Read-Search-Ask</a> si vous êtes bloqué.",
```
Comment (raisedadead) [inline seed/challenges/01-front-end-development-certification/intermediate-ziplines.json]: The wiki link has changed to https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/FreeCodeCamp-Get-Help

File: seed/challenges/01-front-end-development-certification/intermediate-ziplines.json [hunk 3]
```diff
@@ -139,8 +163,22 @@
         "Cuando hayas terminado, pulsa el botón de \"I've completed this challenge\" e incluye un enlace a tu CodePen.",
         "Puedes obtener retroalimentación sobre tu proyecto por parte de otros campistas, compartiendolo en nuestra <a href='//gitter.im/freecodecamp/codereview' target='_blank'>Sala de chat para revisión de código</a>. También puedes compartirlo en Twitter y en el campamento de tu ciudad (en Facebook)."
       ],
+      "descriptionFr": [
+        "<strong>Objectif:</strong> Créer une application sur <a href='http://codepen.io' target='_blank'>CodePen.io</a> qui a un fonctionnement similaire à: <a href='http://codepen.io/FreeCodeCamp/full/pgNRvJ' target='_blank'>http://codepen.io/FreeCodeCamp/full/pgNRvJ</a>.",
```
Comment (raisedadead) [inline seed/challenges/01-front-end-development-certification/intermediate-ziplines.json]: make the `http` links to `https`

File: seed/challenges/01-front-end-development-certification/intermediate-ziplines.json [hunk 4]
```diff
@@ -139,8 +163,22 @@
...
+        "<strong>Règle #1:</strong> Essayez de terminer le projet sans voir le code de l’exemple fourni.",
+        "<strong>Règle #2:</strong> Essayez de répondre aux <a href='https://fr.wikipedia.org/wiki/Récit_utilisateur' target='_blank'>récits d’utilisateurs</a> suivants. Utilisez n’importe quel librairie ou API en ajoutant votre style personnel.",
+        "<strong>Récit d’utilisateur:</strong> Je peux rechercher des articles de Wikipedia dans une barre de recherche et voir les résultats de ma recherche.",
+        "<strong>Récit d’utilisateur:</strong> Je peux cliquer sur un bouton pour voir un article aléatoire.",
+        "<span class='text-info'>Suggestion #1:</span> Vous pouvez utiliser ce lien pour avoir un article aléatoire: <code>http://en.wikipedia.org/wiki/Special:Random</code>.",
```
Comment (raisedadead) [inline seed/challenges/01-front-end-development-certification/intermediate-ziplines.json]: make the `http` links to `https`

File: seed/challenges/01-front-end-development-certification/intermediate-ziplines.json [hunk 5]
```diff
@@ -199,9 +237,25 @@
...
+        "<strong>Suggestion:</strong>Voir un exemple d’appel de l’API JSON de Twitch.tv sur <a href='https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/Front-End-Project-Use-the-Twitchtv-JSON-API' target='_blank'>https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/Front-End-Project-Use-the-Twitchtv-JSON-API</a>.",
+        "<strong> Suggestion:</strong> La documentation de l’exemple d’API est disponible sur: <a href='https://github.com/justintv/Twitch-API/blob/master/v3_resources/streams.md#get-streamschannel' target='_blank'>https://github.com/justintv/Twitch-API/blob/master/v3_resources/streams.md#get-streamschannel</a>.",
+        "<strong> Suggestion:</strong> Voici un tableau d’utilisateurs qui diffusent régulièrement en relation avec la programmation: <code>[\"freecodecamp\", \"storbeck\", \"terakilobyte\", \"habathcx\",\"RobotCaleb\",\"thomasballinger\",\"noobs2ninjas\",\"beohoff\"]</code>",
+        "N’oubliez pas d’utiliser la méthode  <a href='//github.com/FreeCodeCamp/freecodecamp/wiki/How-to-get-help-when-you-get-stuck' target='_blank'>Read-Search-Ask</a> si vous êtes bloqué.",
```
Comment (raisedadead) [inline seed/challenges/01-front-end-development-certification/intermediate-ziplines.json]: Update the wiki link

**Comment by BerkeleyTrue** [comment]
closing as stale. Please feel free to reopen when ready.

**Pull request closed without merging**